app/code/Magento/Pwa/Test/Mftf/Test/CreateTestData.xml

```diff
@@ -81,4 +81,33 @@
             <deleteData createDataKey="createConfigProductAttribute" stepKey="deleteConfigProductAttribute"/>
         </after>
     </test>
+
+    <!--This test just creates simple product with image. This has been extended in actual tests-->
+    <test name="CreateSimpleProductWithImage">
```

Comment:
General question - does this mean these tests are being executed on their own and skewing results a little? Is there any way to mark these as skipped, and then un-skip in the tests that extend them?

Reply:
These tests never gets executed until either extended in a test or explicitly ran using testname or group. Our actual tests have group name "PWA" and these data tests have group name "Data". So we always run PWA to execute Venia tests. In future if we create test suites we just should not add group "Data" in include block.